js/modules/loki_app_dot_net_api.js

```diff
@@ -873,76 +692,33 @@ class LokiAppDotNetServerAPI {
       }
       return null;
     }
-    if (!res.ok) {
+    if (res.statusCode !== 200) {
```

Comment:
Hmm, anything from 200 up to 299 should be considered an OK response, was the server not returning the token on another 2xx?

Reply:
This is how we check for a success for the `get_challenge` endpoint. 
It's the same way for uploading an avatar.
We just need to be consistent with the backend. 

and this is the backend code:
```js
getChallenge(pubKey).then(keyInfo => {
      res.status(200).type('application/json').end(JSON.stringify(keyInfo));
    }).catch(err => {
      console.log(`Error getting challenge: ${err}`);
      res.status(500).type('application/json').end(JSON.stringify({
        error: err.toString(),
      }));
      return;
    });

```

Reply:
Amended the code